activation_service_poc/start.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
-#
-cd ..
-make dockerbuild-go
-cd activation_service_poc
 
-export IMAGE=$(docker images | head -n 2 | tail -n 1 | awk '{print $3}')
+( cd .. && make dockerbuild-go )
```

Comment:
You can avoid changing the directory:
```suggestion
make -C .. dockerbuild-go
```

Btw, what are the parentheses around cd .. && make for?

Reply:
Everything inside of parentheses is executed inside of subshell, so with a separate context which is not changing the main context (whatever you would export/define or if you would change cwd - this changes will be only visible in that subshell). Especially useful when you need to change some env var temporarily only for part of commands.

Good idea to use `-C ..` - i'll adapt my change for it.